css/specials.css

```diff
@@ -1,123 +1,131 @@
+/* Google font assignment */
+
+body {
+	font-family: 'Montserrat', Arial, sans-serif;
+}
+
+.header {
+	background-color: #048004;
+}
+
 h1 {
-    background-color: green;
-    border-style: double;
-    font-family: Georgia, 'Times New Roman', Times, serif;
-    text-align: center;
-    color:white;
-    font-size: 60px;
+	/* background-color: green;
+	border-style: double;
+	font-family: Georgia, 'Times New Roman', Times, serif; */
+	text-align: center;
+	color: white;
+	font-size: 60px;
+	font-weight: 900s;
+	padding-top: 4px;
 }
+
 body {
-    background-color: white;
-    text-align: center;
+	background-color: white;
+	text-align: center;
 }
 
-#socialMedia{
-    text-align: left;
-    position:fixed;
-    width: 30px;
-    bottom: 0;
+#socialMedia {
+	text-align: left;
+	position: fixed;
+	width: 30px;
+	bottom: 0;
 }
 
-
 table, th, td {
-    border: 3px solid black;
-    font-family: cursive;
-    text-align: center;
-    border-color: white;
+	border: 3px solid black;
+	/* font-family: cursive; */
```

Comment:
delete

Reply:
Updated stylesheets to remove unused and commented out styles.